ui/src/app/edge/live/energymonitor/chart/section/consumptionsection.component.ts

```diff
@@ -69,16 +107,41 @@ export class ConsumptionSectionComponent extends AbstractSection {
             topLeft: { x: v, y: v * -1 },
             middleLeft: { x: 0, y: 0 },
             bottomLeft: { x: v, y: v },
-            topRight: { x: r, y: v * -1 },
-            bottomRight: { x: r, y: v },
-            middleRight: { x: r - v, y: 0 }
+            topRight: { x: r * 1.2, y: v * -1 },
+            bottomRight: { x: r * 1.2, y: v },
+            middleRight: { x: r * 1.2 - v, y: 0 }
```

Comment:
Du kannst "r" einfach oben einmal berechnen; oder sinnvoller, da das anscheinend in jeder Section so ist: berechne den größeren "radius" gleich in der AbstractSectionComponent beim Aufruf.